Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.9-slim
+WORKDIR /app
+COPY requirements.txt /app/requirements.txt
+RUN python -m venv venv
```

Comment:
I am unsure if `venv` needs to be initiated when creating a docker image. Do you have a particular reason behind doing so?

Reply:
Having virtual env in docker ensures that there is consistency accors different environment and not have a package conflict

Reply:
But Docker itself creates separate containers for this purpose.

Reply:
Understood , i thought we are working with multiple python version in the same docker , which is not he case